EasyVCR/Censors.cs

```diff
@@ -15,11 +15,10 @@ namespace EasyVCR
     /// </summary>
     public sealed class Censors
     {
-        private readonly List<string> _bodyParamsToCensor;
-        private readonly bool _caseSensitive;
+        private readonly List<CensorElement> _bodyElementsToCensor;
         private readonly string _censorText = "*****";
```

Comment:
Didn't we discuss changing these from stars to the word `REDACTED` to be more explicit?

Reply:
That's set in our implementation of the VCR in our client library. The VCR itself defaults to asterisks, but can be overridden.